### PR TITLE
Minor autolinking template update prep

### PR DESCRIPTION
**Pull request by jonthysell**
This change fixes the cli template so that apps created in the 62 timeframe will work cleanly when autolinking arrives in 63.

Related to #5044.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5090)

### CODE REVIEW

File: vnext/local-cli/generator-windows/templates/cpp/src/AutolinkedNativeModules.g.h [hunk 1]
```diff
@@ -1,17 +1,14 @@
-// AutolinkedNativeModules.g.h -- contents generated by "react-native run-windows"
-
-#include "pch.h"
-
+// AutolinkedNativeModules.g.h contents generated by "react-native autolink-windows"
 #pragma once
 
-// clang-format off
+#include "AutolinkedNativeModules.g.h"
```
Comment (asklar) [inline vnext/local-cli/generator-windows/templates/cpp/src/AutolinkedNativeModules.g.h:4]: is this trying to include itself?

File: vnext/local-cli/generator-windows/templates/cpp/src/AutolinkedNativeModules.g.h [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
 
+// clang-format off
 namespace winrt::Microsoft::ReactNative
 {
 
 static void RegisterAutolinkedNativeModulePackages(winrt::Windows::Foundation::Collections::IVector<winrt::Microsoft::ReactNative::IReactPackageProvider> const& packageProviders)
```
Comment (asklar) [inline vnext/local-cli/generator-windows/templates/cpp/src/AutolinkedNativeModules.g.h:10]: this is providing an implementation but the .g.cpp file has the right one?

Comment (jonthysell) [reply vnext/local-cli/generator-windows/templates/cpp/src/AutolinkedNativeModules.g.h:10]: Whoops.

**Comment by jonthysell** [comment]
I pushed while I was verifying, now it actually builds.

**Comment by NickGerleman** [comment]
Removing the `Backport to 0.62` label, since we need to rename it as part of the deprecation in favor of `Request Backport` which was already added to this PR. Triage of this item should still happen soon.

**Comment by Khalef1** [comment]
This has already been backported, updating tags

**Pull request merged**